notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>I think we should add why people want to use AzureML vs the traditional approach and hightlight its benefits</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I agree here. This is related to the three notebooks [here](https://github.com/Microsoft/Recommenders/blob/jeremr-parallel-top10/notebooks/02_model/parallel_scoring_0_prepare_azure_resources.ipynb). It did require some updates to SARSingleNode to parallelize, but the value to it is that those updates were minor, and then you get scalable performance - you can do process the 20M dataset pretty easily.

Reply:
@jreynolds01 I'm working on clarifying goal from our side. My understanding is we want to create a simple example that shows the minimal code change is required to run a local training algorithm on aml, at least that's one of the goal. Once I get a clear set of goals, I'll put that somewhere

Reply:
Confirmed that we want to show an easy example of how to run a training script on Azure ML. So we don't start of throwing a really complex aml example at user